docs/logs/getting-started-aspnetcore/README.md

```diff
@@ -0,0 +1,98 @@
+# Getting Started with OpenTelemetry .NET Logs in 5 Minutes - ASP.NET Core Application
+
+First, download and install the [.NET
+SDK](https://dotnet.microsoft.com/download) on your computer.
+
+Create a new web application:
+
+```sh
+dotnet new web -o aspnetcoreapp
+cd aspnetcoreapp
+```
+
+Install the
+[OpenTelemetry.Exporter.Console](../../../src/OpenTelemetry.Exporter.Console/README.md)
+and
+[OpenTelemetry.Extensions.Hosting](../../../src/OpenTelemetry.Extensions.Hosting/README.md)
+packages:
+
+```sh
+dotnet add package OpenTelemetry.Exporter.Console
+dotnet add package OpenTelemetry.Extensions.Hosting
+```
+
+Copy the [HelloLogs.cs](./HelloLogs.cs) and [Program.cs](./Program.cs) files to
+the project folder.
+
+Run the application again (using `dotnet run`) and then browse to the URL shown
+in the console for your application (e.g. `http://localhost:5154`). You should see
+the logs output from the console.
+
+```text
+LogRecord.Timestamp:               2023-09-01T23:36:24.3541759Z
+LogRecord.CategoryName:            Microsoft.Hosting.Lifetime
+LogRecord.Severity:                Info
+LogRecord.SeverityText:            Information
+LogRecord.Body:                    Content root path: {contentRoot}
+LogRecord.Attributes (Key:Value):
+    contentRoot: D:\repo\opentelemetry-dotnet\docs\logs\getting-started-aspnetcore\
+    OriginalFormat (a.k.a Body): Content root path: {contentRoot}
+
+Resource associated with LogRecord:
+service.name: getting-started-aspnetcore
+service.instance.id: d4f09b09-6613-4d09-8a2d-1508e4ce9067
+telemetry.sdk.name: opentelemetry
+telemetry.sdk.language: dotnet
+telemetry.sdk.version: 1.6.0-rc.1.16
+
```

Comment:
would be nice to modify consoleexporter separately to put some marker between each LogRecord. Like "======"/similar.

Reply:
Out of the scope of this PR, I think we should really output YAML by default.